client/build.shared

```diff
@@ -0,0 +1,70 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'checkstyle'
+
+repositories {
+  mavenCentral()
+  maven {
+    url "http://oss.sonatype.org/content/groups/public/"
+  }
+}
+
+project.archivesBaseName='litesockets-http-client'
+
+dependencies {
+  testCompile group: 'junit', name: 'junit', version: '4.12'
+
+  compile (
+    "org.threadly:threadly:$threadlyVersion",
+    "org.threadly:litesockets:$litesocketsVersion",
+    "org.threadly:litesockets-http-protocol:0.6.1-SNAPSHOT",
+  )
+  testCompile (
+    "org.threadly:litesockets-http-server:0.6.1-SNAPSHOT",
```

Comment:
We need to make either the client not depend on the server, or the server not depend on the client.  As written currently, there is no way I can upload release artifacts as they would all depend on each other.